video_streaming_ml/common/VideoStream.py

```diff
@@ -0,0 +1,34 @@
+# import the necessary packages
+from imutils.video import WebcamVideoStream
+
+
+class VideoStream:
+	def __init__(self, src=0, usePiCamera=False, resolution=(320, 240), framerate=32):
+		# check to see if the picamera module should be used
+		if usePiCamera:
+			# only import the picamera packages unless we are
+			# explicity told to do so -- this helps remove the
+			# requirement of `picamera[array]` from desktops or
+			# laptops that still want to use the `imutils` package
+            from imutils.video import PiVideoStream
+			# initialize the picamera stream and allow the camera
+			# sensor to warmup
+			self.stream = PiVideoStream(resolution=resolution,
+				framerate=framerate)
```

Comment:
Seems like the indentation is such that this would import regardless of `usePiCamera` but I am not certain